ci-operator/config/openshift/managed-velero-operator/openshift-managed-velero-operator-master.yaml

```diff
@@ -1,24 +1,13 @@
 base_images:
   base:
-    name: ubi
+    name: ubi-minimal
```

Comment:
unrelated?

Reply:
No, fixing oversight to match repo: https://github.com/openshift/managed-velero-operator/blob/6cb4cfec598c7fb80046c4f7ff4d2bb24f501e60/build/Dockerfile#L11

Reply:
Updated PR body to clearly call this out